app/models/user.rb

```diff
@@ -33,4 +33,11 @@ def is_member_or_admin?
     admin? || is_member?
   end
 
+  def is_company_owner?
+    if has_company?
+      company = Company.find(self.company.id)
+      company.company_number == membership_application.company_number
```

Comment:
My above comment is wrong.  this method explicitly checks to see if a user belongs to a specific company.   `has_company` just checks to see if any company has been instantiated.